packages/pyright-internal/src/analyzer/typeEvaluator.ts

```diff
@@ -120,17 +120,17 @@ import {
     VariableDeclaration,
 } from './declaration';
 import {
-    ResolvedAliasInfo,
     createSynthesizedAliasDeclaration,
     getDeclarationsWithUsesLocalNameRemoved,
     getNameNodeForDeclaration,
     resolveAliasDeclaration as resolveAliasDeclarationUtil,
+    ResolvedAliasInfo,
```

Comment:
looks like an eslint difference between pylance and pright.. pyright doesn't have "simple-import-sort/imports" on https://github.com/microsoft/pyrx/pull/1537